cogs/Announce.py

```diff
@@ -278,10 +283,17 @@ async def send(self, ctx):
             embed = self.construct_embed(ctx.guild)
             if self.roles[ctx.guild.id]:
                 for receiver in self.get_receivers(ctx.guild.id, ctx.guild.roles):
-                    await receiver.send(embed=embed)
+                    try:
+                        await receiver.send(embed=embed)
+                    except (discord.Forbidden, AttributeError, discord.HTTPException) as e:
+                        logging.error(f'User {receiver.id} cannot recieve dms')
```

Comment:
Add tests for this please

Reply:
Added test for sending to a specfic role and for failing dms to a specifc role